docs/config/README.md

```diff
@@ -471,23 +471,37 @@ current directory.
 
 ### Options
 
-| Variable          | Default      | Description                                             |
-| ----------------- | ------------ | ------------------------------------------------------- |
-| `conflicted`      | `"="`        | This branch has merge conflicts.                        |
-| `ahead`           | `"⇡"`        | This branch is ahead of the branch being tracked.       |
-| `behind`          | `"⇣"`        | This branch is behind of the branch being tracked.      |
-| `diverged`        | `"⇕"`        | This branch has diverged from the branch being tracked. |
-| `untracked`       | `"?"`        | There are untracked files in the working directory.     |
-| `stashed`         | `"$"`        | A stash exists for the local repository.                |
-| `modified`        | `"!"`        | There are file modifications in the working directory.  |
-| `staged`          | `"+"`        | A new file has been added to the staging area.          |
-| `renamed`         | `"»"`        | A renamed file has been added to the staging area.      |
-| `deleted`         | `"✘"`        | A file's deletion has been added to the staging area.   |
-| `show_sync_count` | `false`      | Show ahead/behind count of the branch being tracked.    |
-| `prefix`          | `[`          | Prefix to display immediately before git status.        |
-| `suffix`          | `]`          | Suffix to display immediately after git status.         |
-| `style`           | `"bold red"` | The style for the module.                               |
-| `disabled`        | `false`      | Disables the `git_status` module.                       |
+| Variable            | Default                    | Description                                                     |
+| ------------------- | -------------------------- | --------------------------------------------------------------- |
+| `conflicted`        | `"="`                      | This branch has merge conflicts.                                |
+| `conflicted_count`  | [link](#git-status-counts) | Show and style the number of conflicts.                         |
```

Comment:
I think this might be easier to read as

| Variable | Default |
| ---| --- |
| `conflicted_count.enabled` | `true` |
| `conflicted_count.style` | `bold red` |

Reply:
Well, it should be OK as it seems to be a common practice to extract shared nested configuation table to a separate passage.